### ISSUE TITLE
Loss stagnates at 1.0 during training on custom dataset 

### ISSUE THREAD

**Issue by LordApplesause** [coqui-ai/TTS#768]
I have been finetuning off a model using Glow TTS on google colab. The training quickly dropped from to 1.0 loss and has increased for the past 1400 steps or so. I made sure to remove any noise or silence from my dataset of 300 samples, and used the analyze spectrograms notebook to check the configurations, so I am lost on what I need to do to make the loss drop further. Do I just need to continue to train?
 
Tensorboard results: 
![image](https://user-images.githubusercontent.com/50555644/131248974-76194832-ca92-4d1d-8078-d0533abeb825.png)

Configuration: 
![image](https://user-images.githubusercontent.com/50555644/131249285-fd1fe65f-2bb3-4428-a7c8-ba4028475a8a.png)

Dataset files: 
https://drive.google.com/drive/folders/1OOIriahYvNPRnK3NMxzH0L1XF_cIGVuM?usp=sharing

Colab notebook: 
https://colab.research.google.com/drive/1vYMd3FBpbpnFZeJlSZWCINdERAS_VUSr?usp=sharing

**Comment by LordApplesause** [comment]
Closing it now, realized I was kinda being a dumbass and was not finetuning off a preexisting model!

**Comment by LordApplesause** [comment]
After training for 1000 epochs on a glow tts pre-trained model of 300,000 steps, I actually ran into the same issue. What else can I do?

**Comment by erogol** [comment]
how does the model sound?

**Comment by erogol** [comment]
I move it to the discussions as it does not conform to the issue template.
